src/private/doctor/js/upcomingVisitsScript.js

```diff
@@ -72,20 +72,22 @@ function fetchEventsAndAddToCalendar() {
         start: '2020-01-17T12:00:00', // a property!
         end: '2020-01-17T14:00:00' // a property! ** see important note below about 'end' **
     }
- * @param eventsJson is server response containting appointments as array of events
+ * @param eventsArray is server response containting appointments as array of event objects
  */
-function addEventsToCallendar(eventsJson) {
+function addEventsToCallendar(eventsArray, calendarInstance) {
     // Iterate through events, process and add to callendar
-    for (let i = 0; i < eventsJson.length; i++) {
-        let jsonEvent = eventsJson[i];
+    for (let i = 0; i < eventsArray.length; i++) {
```

Comment:
for(let i of eventArray) -> Czytelniej i prościej :D 
ogólnie można taką petle w stylu C pisać ale js ma dodatkowo for(let x in object) dla obiektów i for(let x of array) dla tablic :)

Reply:
fakt, zapomniałem, że js też to ma, poprawiam ;)